contrib/build_locale.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Would be nice if the expected arg was checked for being present and validated.

I did not check if this works, but something like this:  
```sh
if [ -z "$1" ] || [ ! -d "$1" ]; then
    echo "usage: $0 locale_path"
    exit 1
fi
```

Reply:
Sure. I don't see the need of the first check - `''` is never a valid directory name anyway.